etc/php-fpm.d/pool.conf

```diff
@@ -24,3 +24,8 @@ pm.status_path = /status
 ping.path = /ping
 
 catch_workers_output = yes
+
+php_admin_value[memory_limit] = 512M
+php_admin_flag[log_errors] = on
+php_admin_value[error_log] = '/proc/self/fd/2'
+php_admin_value[display_errors] = 'stderr'
```

Comment:
most of this variables are now set in a [seperate file](https://github.com/mriedmann/humhub-docker/blob/master/etc/php7/conf.d/99-custom.ini) which also supports the use of ENV-vars for configuration. Setting them here as well could mess with this behavior.

Reply:
Good to know. I've never played with php-fpm before now so I wasn't sure at all where these variables needed to be set. I just stopped iterating when things seemed to be working as I expected.